config/v1/types_registries.go

```diff
@@ -0,0 +1,47 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Registries holds cluster-wide information about how to handle the registries config.  The canonical name is `cluster`
+type Registries struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// spec holds user settable values for configuration
+	Spec RegistriesSpec `json:"spec"`
+	// status holds observed values from the cluster. They may not be overridden.
+	Status RegistriesStatus `json:"status"`
```

Comment:
I am not sure what this is going to be used for.

Reply:
i don't think there's any requirement to have it until/unless we have a use for it.

Reply:
Removed, I don't see any use of it for now